core/pom.xml

```diff
@@ -246,6 +246,20 @@ THE SOFTWARE.
         </exclusion>
       </exclusions>
     </dependency>
+    <!--
+    still including the xpp3 driver to ensure backwards compatibilty
+    for other plugins that may be depending on it
+    -->
+    <dependency>
+      <groupId>xpp3</groupId>
+      <artifactId>xpp3</artifactId>
+      <version>1.1.4c</version>
+    </dependency>
```

Comment:
NIT: I would have put this back at the original place, so that this does *not* show up as a change in this PR. (note: do not spend doing this, just a thought for next time).

Reply:
ack will keep that in mind in the future